### ISSUE TITLE
additional fields for the database

### ISSUE THREAD

**Issue by duncanokeyo** [tonyofrancis/Fetch#195]
could it be possible to include filename and file type in the database fields

**Comment by tonyofrancis** [comment]
@duncanokeyo I can add those fields but the tag field does not work well for you?. What about setting custom json into the tag field. eg::

```java
HashMap<String, String> data = new HashMap<>();
        data.put("","");
        JSONObject jsonObject = new JSONObject(data);
        Request request = new Request("","");
        request.setTag(jsonObject.toString());
```
What if I add just a single field called `dataMap` . That is of type `HashMap<String, String`> . Then you can use it to store file information???

**Comment by duncanokeyo** [comment]
yes, i think that would work

**Comment by tonyofrancis** [comment]
@cc-rock Just published Fetch 2.2.0-RC1. The request class now has a new extras field of type `Map<String, String>` The changelog is huge so you may want to review it. Looking forward to your feedback.
